docs/se/guides/tracing.adoc

```diff
@@ -447,21 +465,19 @@ public class GreetService implements HttpService {
   private void outboundMessageHandler(ServerRequest request, ServerResponse response) {
     Invocation.Builder requestBuilder = webTarget.request();
 
-    // <2>
-    var spanBuilder = request.tracer()
-                .buildSpan("outboundMessageHandler");
-    request.spanContext().ifPresent(spanBuilder::asChildOf);
-    Span span = spanBuilder.start();
+    var spanBuilder = Tracer.global().spanBuilder("outboundMessageHandler");
+    request.context().get(SpanContext.class).ifPresent(sc -> sc.asParent(spanBuilder));
+    var span = spanBuilder.start();     // <2>
 
     try {
       requestBuilder.property(
-          ClientTracingFilter.CURRENT_SPAN_CONTEXT_PROPERTY_NAME, request.spanContext());  // <3>
+          ClientTracingFilter.CURRENT_SPAN_CONTEXT_PROPERTY_NAME, span.context());  // <3>
 
       String result = requestBuilder   // <4>
           .get(String.class);
       response.send(result);
     } finally {
-      span.finish();   // <5>
+      span.end();   // <5>
     }
```

Comment:
This code should probably conform to the earlier example which uses `span.end()` in the `try` block and `span.end(t)` in a `catch (Throwable t)` block.

Reply:
Fixed :)